packages/cors-proxy/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM node:18-alpine
+
+WORKDIR /app
+
+RUN apk add --no-cache \
+    git \
+    # === MDBTools Deps ===
+    sqlite \
+    wget \
+    ca-certificates \
+    autoconf \
+    automake \
+    build-base \
+    glib \
+    glib-dev \
+    libc-dev \
+    libtool \
+    linux-headers \
+    bison flex-dev unixodbc unixodbc-dev \
+    p7zip && \
+    cd /tmp && \
+    wget "https://github.com/brianb/mdbtools/archive/0.7.1.zip" && \
+    unzip 0.7.1.zip && rm 0.7.1.zip && \
+    cd mdbtools-0.7.1 && \
+    autoreconf -i -f && \
+    ./configure --with-unixodbc=/usr/local --disable-man && make && make install && \
+    cd /tmp && \
+    apk del autoconf automake build-base glib-dev libc-dev unixodbc-dev flex-dev && \
+    rm -rf mdbtools-0.7.1
```

Comment:
This can be removed since it's just related to Microsoft Access DB imports (Living Cookbook) in the API server.